deploy/helm/sumologic/templates/deployment.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: apps/v1
```

Comment:
should we rename these files (`deployment.yaml` and `events-deployment.yaml`) ? Are other templates calling these `statefulset.yaml` or something?

Reply:
yes we should